src/drafts/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -227,7 +228,7 @@ const MarkdownEditor = forwardRef<MarkdownEditorHandle, MarkdownEditorProps>(
         ({
           focus: opts => inputRef.current?.focus(opts),
           scrollIntoView: opts => containerRef.current?.scrollIntoView(opts),
-        }) as MarkdownEditorHandle,
+        } as MarkdownEditorHandle),
```

Comment:
```suggestion
        }) as MarkdownEditorHandle,
```

somehow this throws linting errors in CI but locally it autocorrects to the prev 🤷